lib/semantics/resolve-names.cc

```diff
@@ -925,12 +936,22 @@ class ConstructVisitor : public DeclarationVisitor {
   void Post(const parser::ExitStmt &x) { CheckRef(x.v); }
 
 private:
-  // This represents: associate-name => expr | variable
-  // expr is set unless there were errors
+  // R1105 selector -> expr | variable
+  // expr is set in either case unless there were errors
+  struct Selector {
+    Selector() : variable{nullptr} {}
```

Comment:
It might be more robust to declare `variable` below with `nullptr` as its default initializer.

Reply:
Thanks for the suggestions, I've incorporated all of them.